test/BuyTshirt.spec.ts

```diff
@@ -22,30 +22,42 @@ describe('Buy a t-shirt', () => {
   const signInStepPage: SignInStepPage = new SignInStepPage();
   const summaryStepPage: SummaryStepPage = new SummaryStepPage();
 
-  beforeAll(async () => {
-    await browser.get(webpage);
-
-    await menuContentPage.goToTShirtMenu();
-    await productListPage.goToTShirtDetail();
-    await productDetailPage.goToTShirtModal();
-    await productAddedModalPage.goToSummary();
-    await summaryStepPage.goToSignIn();
-
-    await signInStepPage.sendMail(email);
-    await signInStepPage.sendPassword(password);
-    await signInStepPage.signIn();
-
-    await addressStepPage.goToShipping();
-
-    await shippingStepPage.conditionsAgree();
-    await shippingStepPage.goToPayment();
-
-    await paymentStepPage.goToBank();
-    await bankPaymentPage.goToOrderResume();
-  });
-
-  it('then should be bought a t-shirt', async () => {
-    await expect(orderResumePage.order())
-   .toBe('Your order on My Store is complete.');
+  describe('When the webpage gets opened', () => {
+    beforeAll(async() => {
+      await browser.get(webpage);
+    });
+
+    describe('and I start the t-shirt buying process', () => {
```

Comment:
para llevarlo más a lenguaje de negocio podrías poner `and I decide to buy a t-shirt`

Reply:
Ok, ya mismo lo corrijo